data/protein_quantification.csv

```diff
@@ -0,0 +1,27 @@
+,u87pi norm,h5m norm,plate
```

Comment:
Having a blank name for this first column is not good. Not sure if you create this file or not